### PR TITLE
add kubernetes system containers

### PR DESCRIPTION
**Pull request by jasonbrooks**
These are the kubernetes system containers I've blogged about at http://www.projectatomic.io/blog/2017/05/system-containerized-kube/

### CODE REVIEW

**Comment by strigazi** [comment]
lgtm

File: kubernetes-apiserver/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM kubernetes-master:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV container=docker
+
+ENV NAME=kubernetes-apiserver VERSION=0.1 RELEASE=8 ARCH=x86_64
+LABEL BZComponent="$NAME" \
```
Comment (ashcrow) [inline kubernetes-apiserver/Dockerfile]: My understanding is that labels for system containers should be lower case.

File: kubernetes-controller-manager/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM kubernetes-master:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV container=docker
+
+ENV NAME=kubernetes-controller-manager VERSION=0.1 RELEASE=8 ARCH=x86_64
+LABEL BZComponent="$NAME" \
```
Comment (ashcrow) [inline kubernetes-controller-manager/Dockerfile]: Same label note.

File: kubernetes-kubelet/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
+FROM kubernetes-node:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV container=docker
+
+ENV NAME=kubernetes-kubelet VERSION=0 RELEASE=8 ARCH=x86_64
+LABEL BZComponent="$NAME" \
```
Comment (ashcrow) [inline kubernetes-kubelet/Dockerfile]: Same label note.

File: kubernetes-master/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM registry.fedoraproject.org/fedora:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV NAME=kubernetes-master VERSION=0 RELEASE=0 ARCH=x86_64
+LABEL BZComponent="$NAME" \
```
Comment (ashcrow) [inline kubernetes-master/Dockerfile]: Same label note.

File: kubernetes-node/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,12 @@
+FROM registry.fedoraproject.org/fedora:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV NAME=kubernetes-node VERSION=0 RELEASE=0 ARCH=x86_64
+LABEL BZComponent="$NAME" \
```
Comment (ashcrow) [inline kubernetes-node/Dockerfile]: Same label note.

File: kubernetes-proxy/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,25 @@
+FROM kubernetes-node:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV container=docker
+
+ENV NAME=kubernetes-proxy VERSION=0 RELEASE=8 ARCH=x86_64
+LABEL BZComponent="$NAME" \
```
Comment (ashcrow) [inline kubernetes-proxy/Dockerfile]: Same label note.

File: kubernetes-scheduler/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,22 @@
+FROM kubernetes-master:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV container=docker
+
+ENV NAME=kubernetes-scheduler VERSION=0.1 RELEASE=8 ARCH=x86_64
+LABEL BZComponent="$NAME" \
```
Comment (ashcrow) [inline kubernetes-scheduler/Dockerfile]: Same label note.

**Comment by jasonbrooks** [comment]
What else is needed for this one to be merged?

**Comment by ashcrow** [comment]
@jasonbrooks double checking but I don't think anything else is needed.

**Comment by ashcrow** [comment]
@rh-atomic-bot r+

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ b67f561

**Comment by giuseppe** [comment]
I think Homu doesn't like r+ without the hash specified

**Pull request closed without merging**